install

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env php
```

Comment:
Since the current update command is in the chromedriver-bin directory, maybe this one should be as well?

Reply:
I opted to go with a general `bin/` dir so we can remove both `chromedriver-bin/` and `geckodriver-bin/` later and have the files in `bin/chromedriver` and `bin/geckodriver`